projects/organization-management/src/app/pages/users-edit-profile/users-edit-profile-page.component.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { MockComponent } from 'ng-mocks';
+
+import { coreReducers } from 'ish-core/store/core-store.module';
+import { ngrxTesting } from 'ish-core/utils/dev/ngrx-testing';
+import { LoadingComponent } from 'ish-shared/components/common/loading/loading.component';
+
+import { UserProfileFormComponent } from '../../shared/user/user-profile-form/user-profile-form.component';
```

Comment:
this has a linting issue related with the shared folder, since we will have more for permissions, roles and budget I left like this so you can decide the best approach. If needed I can apply the necessary changes on monday

Reply:
disabled the linting rule for now